ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-CARM-nonprod.json

```diff
@@ -13,6 +13,21 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "CUI_UPLOAD_MEDIATION_DOCUMENTS",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "APP-SOL-SPEC-PROFILE",
+          "RES-SOL-ONE-SPEC-PROFILE",
+          "RES-SOL-TWO-SPEC-PROFILE",
+          "CITIZEN-DEFENDANT-PROFILE"
```

Comment:
should this be CRU for citzen and R for LRs? if it has CRU for LRs it'll show up in the xui dropdown i think?

Reply:
thank you.
Done the change